gallery/gallery/lib/studies/shrine/app.dart

```diff
@@ -78,6 +78,18 @@ class _ShrineAppState extends State<ShrineApp> with TickerProviderStateMixin {
     );
   }
 
+  // Closes the bottom sheet if it is open (this should only be relevant for Android).
```

Comment:
nit, suggestion: would remove the () about Android

Reply:
Done!